src/main/kotlin/no/nav/syfo/tilgang/TilgangService.kt

```diff
@@ -207,6 +207,16 @@ class TilgangService(
         return tilgang
     }
 
+    suspend fun filterIdenterByVeilederAccess(callId: String, token: Token, personidenter: List<String>): List<String> {
+        return personidenter.filter { personident ->
+            hasTilgangToPerson(
```

Comment:
Trenger kanskje ikke endre på dette nå, men for meg høres det ut som `hasTilgangToPerson` er noe som returnerer `true` hvis `erGodkjent` er `true`. Burde den heller hete `getTilgangToPerson` e.l.?

Reply:
Godt poeng, den returnerer jo tross alt et Tilgang-objekt som kan være begge deler! Jeg lager en lapp på det 📓